community/releases/support-policy.md

```diff
@@ -10,19 +10,53 @@ redirect_from: /docs/support-policy
 
 ## Bugs and Feature Requests
 
-If you have what looks like a bug, or you would like to make a feature request, please use the [GitHub issue tracking system](https://github.com/spinnaker/spinnaker/issues).
+If you have what looks like a bug, or you would like to make a feature request,
+please use the [GitHub issue tracking
+system](https://github.com/spinnaker/spinnaker/issues).
 
-Before you file an issue, please search existing issues to see if your issue is already covered.
+Some things to keep in mind:
+
+* Please search existing issues to see if your issue is already covered. If it
+  is, and you have any new information, please share it! It will help us close
+  the issue faster.
+
+* If you are filing a new issue, please provide as much background information
+  as possible! At the very least, provide the version of Spinnaker you are
+  running, and what steps you took to reproduce the issue. The more information
+  we have, the faster we can fix the problem.
 
 ## Q&A
 
-If you have a question that isn't answered by our documents and FAQ pages, please reach out for help on either Stack Overflow or Slack.
+If you have a question that isn't answered by our documents and FAQ pages,
+please reach out for help on Discourse, Stack Overflow, or Slack.
```

Comment:
Even though they're listed and linked just below, might be worth it to link these here in the paragraph?

Reply:
sg